### PR TITLE
Add skriptfabrik feature collection

### PR DESCRIPTION
**Pull request by schroedan**
## What type of PR is this?

- [x] Add a new dev container collection
- [ ] Update to an existing dev container collection
- [ ] Documentation/spec update
- [ ] Other containers.dev site update (UX, layout, etc)

## Description

This PR is about to add the skriptfabrik dev container features to the list.

### Collection checklist
_If your PR contributes a new collection, please utilize this checklist:_
- [x] Collection name: Dev Container Features by skriptfabrik
- [x] Maintainer name: skriptfabrik
- [x] Maintainer contact link (i.e. link to a GitHub repo, email): https://github.com/skriptfabrik/
- [x] Repository URL: https://github.com/skriptfabrik/devcontainer-features/
- [x] OCI Reference: ghcr.io/skriptfabrik/devcontainer-features
- [x] I acknowledge that this collection provides new functionality, distinct from the existing collections part of this index.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -768,3 +768,8 @@
   contact: https://github.com/duduribeiro/devcontainer-features/issues
   repository: https://github.com/duduribeiro/devcontainer-features/
   ociReference: ghcr.io/duduribeiro/devcontainer-features  
+- name: Dev Container Features by skriptfabrik
+  maintainer: skriptfabrik
+  contact: https://github.com/skriptfabrik/devcontainer-features/issues
+  repository: https://github.com/skriptfabrik/devcontainer-features/
+  ociReference: ghcr.io/skriptfabrik/devcontainer-features
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:775]: Hi 👋

Can you help change the visibility of this "metadata" package (and your Feature package) to public? The [index](https://containers.dev/features) won't be able to crawl the Features in `skriptfabrik/devcontainer-features` if it's private.

Feel free to look at https://containers.dev/guide/author-a-feature#publishing for information.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thank you for the addition, left one comment.

File: _data/collection-index.yml (hunk 1 above)
Comment (schroedan) [reply _data/collection-index.yml:775]: Hello, I have set the package `skriptfabrik/devcontainer-features` to public. Sorry for the inconvenience.

**Review by samruddhikhandale — APPROVED** [review]
Thank you 🎉

**Pull request merged**